packages/dataviews/src/view-list.js

```diff
@@ -67,70 +198,29 @@ export default function ViewList( {
 	}
 
 	return (
-		<ul className="dataviews-view-list">
+		<Composite
+			id={ baseId }
+			render={ <ul /> }
+			className="dataviews-view-list"
+			role="grid"
+			store={ store }
+		>
 			{ usedData.map( ( item ) => {
+				const id = getItemDomId( item );
```

Comment:
Why not using `getItemId` directly? By definition, that's each item's unique identifier.

Reply:
Is this something related to how Composite/Ariakit works? Otherwise, I don't follow what the ids are for at this level. Judging by what I see at runtime, I presume this is to make sure the `id` don't conflict with anything else. We could go simpler and do: `list-item-${getItemId( item )}`.

Reply:
I'm largely unfamiliar with Ariakit's Composite, but I find it weird that it 1) uses DOM ids to track the items and 2) the concept of a store attached to a component.

I understand that, by using Ariakit's Composite, we want to hide a certain layer of complexity (handle onClick/onKeydown events + tab/arrow-key mechanics) though I'd argue that complexity is familiar for most people while the complexity added by Ariakit's Composite mechanics is unfamiliar for most. In terms of maintenance and evolution of this code, I feel a bit uneasy about this trade-off.

Reply:
Asked around to some folks and others think this is fine.